conjureup/juju.py

```diff
@@ -559,16 +513,17 @@ def get_accounts():
     raise Exception("Unable to find accounts")
 
 
-def model_by_owner(user):
+def model_by_owner(controller, user):
```

Comment:
it looks like model_by_owner isn't used anywhere, and it's a little odd - it'll just return the arbitrary first model owned by 'user'.

What do you think about just getting rid of it?